src/_includes/core-nodes/split-use-case.md

```diff
@@ -69,26 +65,22 @@ Keep the `Split using` set to `\n`, which is the line ending character in Unix
 based systems. In our next example, we have a list of 3 cities in Europe, which
 we need individually. 
 
-![Splitting text by line](./images/split-node-by-line.png "Node-RED flow to split text by line")
-
 Note we use the template node here simply, as the "Inject" node does not allow you
 to create multi-line strings.
 
-```json
+{% renderFlow %}
 [{"id":"39a0a053a3696cd7","type":"inject","z":"2862bf5c278ff5bd","name":"Trigger","props":[],"repeat":"","crontab":"","once":false,"onceDelay":0.1,"topic":"","x":130,"y":220,"wires":[["60bf012438abb4eb"]]},{"id":"4b56a3ed831df59e","type":"split","z":"2862bf5c278ff5bd","name":"Split by line","splt":"\\n","spltType":"str","arraySplt":1,"arraySpltType":"len","stream":false,"addname":"","x":470,"y":220,"wires":[["31f8ca22882b297f"]]},{"id":"31f8ca22882b297f","type":"debug","z":"2862bf5c278ff5bd","name":"Print each line","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"payload","targetType":"msg","statusVal":"","statusType":"auto","x":660,"y":220,"wires":[]},{"id":"60bf012438abb4eb","type":"template","z":"2862bf5c278ff5bd","name":"Data in lines","field":"payload","fieldType":"msg","format":"handlebars","syntax":"mustache","template":"Amsterdam\nAndorra la Vella\nAthens","output":"str","x":290,"y":220,"wires":[["4b56a3ed831df59e"]]}]
-```
+{% endrenderFlow %}
 
 #### Splitting by word
 
 When you've split something by line, you might need to split by word. Simply
 putting a space in the `Split using` setting. That doesn't show a visible character
 in the form, as opposed to the line ending character.
 
-![Node-RED split by word](./images/split-node-by-word.png "Split text by space in Node-RED")
-
-```json
+{% renderFlow 400 %}
```

Comment:
@Yndira-E here's an example where height is set

Reply:
Thanks Joe!!
I tried to make them all the same height, but sometimes the flows aren't centered or even visible... I'm guessing that's the reason some have an unusual height.
<img width="624" alt="Screenshot 2024-04-30 at 16 47 04" src="https://github.com/FlowFuse/website/assets/129537638/d03f2aa0-574d-4f49-9b9e-e8ae5a3b28e4">

Maybe some information regarding how to set the height should be added to handbook as well?

Reply:
Yeah, that's caused by https://github.com/FlowFuse/flow-renderer/issues/28